### ISSUE TITLE
Support forms

### ISSUE THREAD

**Issue by thebetterjort** [diegomura/react-pdf#42]
Would like to be able to graba data from PDF forms.

**Comment by diegomura** [comment]
Thanks @thebetterjort . This is definitely something we will like to tackle (if possible) in a near future

**Comment by marteloge** [comment]
I would love this feature!
Have someone been working on this feature? :)

**Comment by thebetterjort** [comment]
Never tried to get it working.

**Comment by valstu** [comment]
I guess react-pdf won't support pulling data out of existing pdfs but I think we could now add support for Acro forms (adding dynamic input fields to pdfs) since support for that was added on pdfkit on this PR https://github.com/foliojs/pdfkit/pull/1002

I would love to participate adding this functionality but tbh don't have much experience working with custom renderers. If someone could push me into right direction I could take a look at this. @diegomura do you see any instant blockers why Acroforms wouldn't work with react-pdf?

**Comment by diegomura** [comment]
Hey @valstu ! Not at all. Everything should be already in place for forms to work. I recently added those primitives to our pdfkit fork. I didn't do it yet because of lack of time 😅  Yesterday I updated the [CONTRIBUTING](https://github.com/diegomura/react-pdf/blob/master/.github/CONTRIBUTING.md) document with some hints about how to setup a react-pdf dev environment. I think the only "blocker" is deciding how forms primitives will look like. I lean to copying react-natives (or react-primitives) as close as possible because that means the learning curve for people about how to use them get's smaller

**Comment by sepsol** [comment]
This would be a great feature, any updates?

**Comment by diego-hourly** [comment]
Yup I'm requiring a feature like this one. Is there a chance for me to help on this? I need it 😆

**Comment by anthares-dev** [comment]
Any update about this functionality?

**Comment by joziasmartini** [comment]
Are there any updates?

**Comment by diegomura** [comment]
Will be added in next version
